File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/service/invocation/entity/PostmanDubboRequest.java
package org.javamaster.invocationlab.admin.service.invocation.entity;

import lombok.Getter;
import lombok.Setter;

/**
 * @author yudong
 */
@Setter
@Getter
public class PostmanDubboRequest implements PostmanRequest {
    String dubboParam;
    String cluster;
    String serviceName;
    String group;
    String interfaceName;
    String version;
    String methodName;
    String dubboIp;

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/service/invocation/entity/PostmanDubboRequest.kt
package org.javamaster.invocationlab.admin.service.invocation.entity

/**
 * @author yudong
 */
class PostmanDubboRequest : PostmanRequest {
    var dubboParam: String? = null
    var cluster: String? = null
    var serviceName: String? = null
    var group: String? = null
    var interfaceName: String? = null
    var version: String? = null
    var methodName: String? = null
    var dubboIp: String? = null
}
